docs/faq.txt

```diff
@@ -164,3 +164,20 @@ Yes! For an example of this check out the :download:`mnist-GAN
 automatically configures the location for writing TensorBoards. Users can also
 directly use ``torch.utils.tensorboard.SummaryWriter`` as shown in the snippet
 above.
+
+
+Scheduling
+----------
+
+How can I control which agents a task is scheduled on?
```

Comment:
Maybe phrase this more in the Determined novice way -- something like
"My cluster has machines with two different types of GPUs, how can I run a task on a specific machine?"